iceoryx_platform/win/include/iceoryx_platform/unistd.hpp

```diff
@@ -22,11 +22,17 @@
 
 #include <io.h>
 #include <process.h>
+#include <stdio.h>
 #include <vector>
 
 #define IOX_SEEK_SET SEEK_SET
 #define _SC_PAGESIZE 1
-#define STDERR_FILENO 2
+#ifndef STDOUT_FILENO
```

Comment:
From https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/fileno?view=msvc-170:
"If stdout or stderr is not associated with an output stream (for example, in a Windows application without a console window), the file descriptor returned is -2."
Do we want this behavior?

Reply:
On the other hand, the man page is not clear regarding of locking of `fileno` calls. Therefore I revert the change and just use the numbers